tasks/kubectl-run-performance-tests.yml

```diff
@@ -0,0 +1,54 @@
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: eu.gcr.io/census-gcr/gcloud-kubectl
+
+params:
+  SERVICE_ACCOUNT_JSON:
+  GCP_PROJECT_NAME:
+  KUBERNETES_CLUSTER:
+  PERFORMANCE_TESTS_IMAGE:
+
+inputs:
+- name: performance-tests-repo
+
+run:
+  path: bash
+  args:
+    - -exc
+    - |
+      cat >~/gcloud-service-key.json <<EOL
+      $SERVICE_ACCOUNT_JSON
+      EOL
+
+      # Use gcloud service account to configure kubectl
+      gcloud auth activate-service-account --key-file ~/gcloud-service-key.json
+      gcloud container clusters get-credentials ${KUBERNETES_CLUSTER} --zone europe-west2 --project ${GCP_PROJECT_NAME}
+
+      SFTP_TARGET_DIRECTORY=$(kubectl get configmap project-config -o=jsonpath="{.data.sftp-target-directory}")
```

Comment:
I think the variable is unused, `.data.sftp-target-directory` doesn't exist anymore in any case.

Reply:
@AdamHawtin probably needs removing from the acceptance tests task as well then.